pkg/platform/authentication/auth.go

```diff
@@ -366,7 +366,17 @@ func (s *Auth) CreateToken() error {
 		return err
 	}
 
-	err = s.cfg.Set(ApiTokenConfigKey, token)
+	err = s.SaveToken(token)
+	if err != nil {
+		return errs.Wrap(err, "SaveToken failed")
+	}
+
+	return nil
+}
+
+// SaveToken will save an API token
+func (s *Auth) SaveToken(token string) error {
```

Comment:
Did you intentionally want to expose "SaveToken" outside the package? Or should it be private?

Reply:
Yes, it's used in the authlet.